src/components/page/DrawerItemsWithScrollspy.tsx

```diff
@@ -38,22 +38,30 @@ const styles = (theme: Theme) => createStyles({
     },
     oldVersionLinkButton: {
         margin: "5%"
+    },
+    isCurrent: {
+        fontWeight: "bolder",
+        color: "#770077",
+        backgroundColor: "#dadaff"
     }
 });
 
 interface DrawerItemsWithScrollspyProps extends WithStyles<typeof styles> {
     items: string[],
 }
 
+const EMPIRICAL_OFFSET = -80;
+
 function DrawerItemsWithScrollspy(props: DrawerItemsWithScrollspyProps) {
     const {items, classes} = props;
 
     return (
         <React.Fragment>
             <Scrollspy
-                items={items}
-                currentClassName="is-current"
+                items={items.map(item => item.toLowerCase())}  // TODO: refactor this
```

Comment:
To be refactored as part of the re-architecture of the whole portfolio application